src/Webhooks.ts

```diff
@@ -78,8 +78,10 @@ const Webhook: WebhookObject = {
       cryptoProvider
     );
 
-    // @ts-ignore
-    const jsonPayload = JSON.parse(payload);
+    const jsonPayload =
+      payload instanceof Uint8Array
+        ? JSON.parse(new TextDecoder().decode(payload))
```

Comment:
I added this condition with the Uint8Array tests since `JSON.parse` does not work as intended on `Uint8Array`